### ISSUE TITLE
Document and implement backup

### ISSUE THREAD

**Issue by mrjones-plip** [synshop/ShopIdentifyer#17]
before we can start using this in production, we need to clearly document how we backup:
- uploads
- unique configs
- mysql DBs

And, as well, document the process to restore a backup.  Then we need to implement it and, of course, test restore process ;)

**Comment by Ths2-9Y-LqJt6** [comment]
I'm thinking this could be done in a single script which just accepts the path you want to backup to: `backup.py /var/backups/Mar.3.2016/`.  It would then read in the `config.py` values so it can do the DB dump, know where the uploaded files are and of course know the config values themselves.

thoughts?

**Comment by munroebot** [comment]
Makes sense to me.  Lets create a script/ directory and start putting stuff like that in there?

I do have a script I was using to back up the database, but if we are going to have a bunch of bootstrap / maintenance utilities then maybe it makes sense to move it.

https://github.com/synshop/ShopIdentifyer/blob/master/sql/db_backup.sh

**Comment by Ths2-9Y-LqJt6** [comment]
sure, `script/` or `scripts/` works! 

the [db_backup](https://github.com/synshop/ShopIdentifyer/blob/master/sql/db_backup.sh).sh is just doing a mysql dump and is hard coded though. what I was envisioning was something that would dynamically read in the values to connect to the DB from the config file.

**Comment by munroebot** [comment]
Closing for now, will revisit in a new Issue
